src/test/ui/error-codes/E0401.rs

```diff
@@ -11,7 +11,7 @@
 trait Baz<T> {}
 
 fn foo<T>(x: T) {
-    fn bar<U, V: Baz<U>, W: Fn()>(y: T) { //~ ERROR E0401
+    fn bfnr<U, V: Baz<U>, W: Fn()>(y: T) { //~ ERROR E0401
```

Comment:
This function is called below, so you need to rename the call, too. Or just undo the rename